build.go

```diff
@@ -189,8 +219,8 @@ func (cmd *buildCommand) Run(args []string) (err error) {
 		frontendAttrs["label:"+kv[0]] = kv[1]
 	}
 
-	fmt.Printf("Building %s\n", initialTag)
-	fmt.Println("Setting up the rootfs... this may take a bit.")
+	fmt.Fprintf(os.Stderr, "Building %s\n", initialTag)
+	fmt.Fprintln(os.Stderr, "Setting up the rootfs... this may take a bit.")
```

Comment:
this is fine, but probably it is also possible to use progresswriter for these messages
https://github.com/docker/buildx/blob/3f18b659a09804c738226dbf6bacbcae54afd7c6/driver/docker-container/driver.go#L31

Reply:
I'm not entirely sure how I would implement that. Can we use `showProgress` with a fabricated StatusResponse with a single Vertex?